modules/osdk-installing-cli.adoc

```diff
@@ -146,9 +147,9 @@ You can install the SDK CLI using Homebrew.
 .Prerequisites
 
 - link:https://brew.sh/[Homebrew]
-- link:https://docs.docker.com/install/[`docker`] v17.03+
-- OpenShift CLI (`oc`) v4.1+ installed
-- Access to a cluster based on Kubernetes v1.11.3+
+- link:https://docs.docker.com/install/[`docker`] v17.03+, link:https://github.com/containers/libpod/blob/master/install.md[`podman`] v1.2.0+, or link:https://github.com/containers/buildah/blob/master/install.md[`buildah`] v1.7+
```

Comment:
There are a couple of links to Github repos. We might need to get approval for those.

Reply:
Made the links only appear for the OKD distro.